src/main/java/util/FileManager.java

```diff
@@ -0,0 +1,96 @@
+package util;
+
+import task.*;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.Serializable;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
good organisation of packages! easy to understand what you are using in this class!